index.js

```diff
@@ -50,9 +56,11 @@ Plugin.prototype.package = function() {
     self.crx.pack().then(function(buffer) {
       mkdirp(self.outputPath, function(err) {
         if (err) throw(err)
-        var updateXML = self.crx.generateUpdateXML();
-        fs.writeFile(self.updateFile, updateXML);
-        fs.writeFile(self.crxFile, buffer);
+    	fs.writeFileSync(self.crxFile, buffer);
+		  if(self.generateUpdateFile) {
+        	var updateXML = self.crx.generateUpdateXML();
+        	fs.writeFileSync(self.updateFile, updateXML);
+        }
```

Comment:
Looks like your whitespace is off 😊 

```suggestion
    	fs.writeFileSync(self.crxFile, buffer);
        if(self.generateUpdateFile) {
          var updateXML = self.crx.generateUpdateXML();
          fs.writeFileSync(self.updateFile, updateXML);
        }
```